source/common/http/conn_manager_impl.h

```diff
@@ -387,9 +388,26 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     NullRouteConfigUpdateRequester() = default;
   };
 
+  class FilterManagerCallbacks {
```

Comment:
class comments for sure, and bonus for commenting what the actual functions should result in